jujugui/static/gui/src/app/components/deployment-flow/signup/test-signup.js

```diff
@@ -144,7 +128,9 @@ describe('DeploymentSignup', function() {
     });
   });
 
-  it('can export the model', function() {
+  xit('can export the model', function() {
```

Comment:
I imagine this is still a WIP but might as well just delete this test instead of skipping it.